### PR TITLE
Updated ONBOARDING to reflect team changes

### PR DESCRIPTION
**Pull request by pdavidc**
- Incorporated Bintray and NPM into project resources and developer accounts
- Added initial section on Agile development workflow
- Added initial section on Development Guidelines

Closes NASAWorldWind/WebWorldWind#150

### CODE REVIEW

**Comment by pdavidc** [comment]
@zglueck This document is not directly related to the website, and may be better suited for a wiki. Until we establish a convention for project-wide developer documentation, I'll be keeping this document up to date.

File: ONBOARDING.md [hunk 1]
```diff
@@ -1,40 +1,67 @@
...
 - [GitHub](https://github.com) account with two-factor authentication enabled
+- [Bintray](https://bintray.com/signup/oss) account (sign up with GitHub)
+- [NPM](https://www.npmjs.com/signup) account
 - [Skype](https://login.skype.com/account/signup-form) account
 - [JetBrains](https://account.jetbrains.com/login) account
-- [World Wind Forum](http://forum.worldwindcentral.com/register.php) account (NASA interns skip this step)
+- [Forum](http://forum.worldwindcentral.com/register.php) account (NASA interns skip this step)
 ... Set Avatar to NASA logo
 ... Set User Title to 'WW Team Member'
 ... Set Biography to 'NASA WorldWind Team Member'
```
Comment (zglueck) [inline ONBOARDING.md]: Are these supposed to be indented bullets?

**Review by zglueck — COMMENTED** [review]
Looks good, just wondering about the `...` for the forum.

File: ONBOARDING.md (hunk 1 above)
Comment (pdavidc) [reply ONBOARDING.md]: They are indeed. Is there a better way with Markdown?

Comment (zglueck) [reply ONBOARDING.md]: I believe two spaces before the dash space will work:
```
- Topic 1
   - Sub Topic A
   - Sub Topic B
- Topic 2
```

- Topic 1
  - Sub Topic A
  - Sub Topic B
- Topic 2

**Comment by pdavidc** [comment]
I've corrected the Markdown syntax for the indented lists.

**Review by zglueck — APPROVED** [review]
This is becoming a great resource, even for existing team members.

**Pull request merged**